client/post-editor/editor-notice/index.jsx

```diff
@@ -84,135 +85,59 @@ export class EditorNotice extends Component {
 				);
 
 			case 'publishFailure':
-				if ( 'page' === type ) {
-					return translate( 'Publishing of page failed.' );
-				}
-
-				if ( 'post' !== type && typeLabel ) {
-					return translate( 'Publishing of %(typeLabel)s failed.', { args: { typeLabel } } );
-				}
-
-				return translate( 'Publishing of post failed.' );
+				return translate( 'Publishing of %(typeLabel)s failed.', {
+					args: { typeLabel: typeLabel.toLowerCase() },
+				} );
 
 			case 'saveFailure':
 				return translate( 'Saving of draft failed.' );
 
 			case 'trashFailure':
-				if ( 'page' === type ) {
-					return translate( 'Trashing of page failed.' );
-				}
-
-				if ( 'post' !== type && typeLabel ) {
-					return translate( 'Trashing of %(typeLabel)s failed.', { args: { typeLabel } } );
-				}
-
-				return translate( 'Trashing of post failed.' );
+				return translate( 'Trashing of %(typeLabel)s failed.', {
+					args: { typeLabel: typeLabel.toLowerCase() },
+				} );
 
 			case 'published':
 				if ( ! site ) {
-					if ( 'page' === type ) {
-						return translate( 'Page published!' );
-					}
-
-					if ( 'post' !== type && typeLabel ) {
-						return translate( '%(typeLabel)s published!', { args: { typeLabel } } );
-					}
-
-					return translate( 'Post published!' );
+					return translate( '%(typeLabel)s published!', { args: { typeLabel: typeLabel } } );
 				}
 
 				if ( 'page' === type ) {
-					return translate( 'Page published on {{siteLink/}}! {{a}}Add another page{{/a}}', {
-						components: {
-							siteLink: (
-								<a
-									href={ site.URL }
-									target="_blank"
-									rel="noopener noreferrer"
-									onClick={ this.handlePillExternalClick }
-								>
-									{ site.title }
-								</a>
-							),
-							a: (
-								<a href={ `/page/${ site.slug }` } onClick={ this.handlePillAddPagePromptClick } />
-							),
-						},
-						comment:
-							'Editor: Message displayed when a page is published, with a link to the site it was published on.',
-					} );
-				}
-
-				if ( 'post' !== type && typeLabel ) {
-					return translate( '%(typeLabel)s published on {{siteLink/}}!', {
-						args: { typeLabel },
+					return translate( 'Page published on {{pageLink/}}! {{a}}Add another page{{/a}}', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **19** times:
`translate( 'Page published on {{siteLink/}}! {{a}}Add another page{{/a}}' )` *ES Score: 10*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).